examples/test.py

```diff
@@ -3,20 +3,20 @@
 import sys, os
 sys.path.append("../")
 
-from zfit.core import optimisation as opt
+from zfit.core import optimization as opt
 from zfit.core import kinematics as kin
 from zfit.physics import constants as const
 from zfit.physics.flavour.rare_decays.bToKstarll import decay_rates as dec
 
 
-if __name__ == "__main__" : 
+if __name__ == "__main__" :
 
   phsp = kin.FourBodyAngularPhaseSpace(1.,8.,)
 
-  
+
   ### Start of model description
 
-  def model(x) : 
+  def model(x) :
```

Comment:
We should also cleanup whitespaes before ":", but I guess we can autopep?

Reply:
Yes and no: autopep or similar for sure where it works (I'll do that) but there are places where I think it is a design decision. Things like:

```
a152 = (100, 1  )
a9   = (1  , 100)
etc.
```
are not usual pep8 style, though probably useful. Any opinion on that, @apuignav ?